integrationtest/backend/lucene/src/test/java/org/hibernate/search/integrationtest/backend/lucene/search/LuceneSearchMultiIndexIT.java

```diff
@@ -63,9 +71,19 @@
 	private IndexMapping_1_2 indexMapping_1_2;
 	private StubMappingIndexManager indexManager_1_2;
 
+	@Parameters(name = "Lucene directory type {0}")
+	public static Object[] data() {
+		return new Object[] { "local-heap", "local-filesystem" };
+	}
+
+	public LuceneSearchMultiIndexIT(String directoryType) {
+		this.directoryType = directoryType;
+	}
+
 	@Before
 	public void setup() {
-		setupHelper.start()
+		setupHelper.start( BACKEND_1_NAME )
+				.withBackendProperty( BACKEND_1_NAME, "directory.type", directoryType )
```

Comment:
You no longer need to define and pass a backend name. You can call `start()` and `withBackendProperty( "directory.type", directoryType )`.